include/ua_types.h

```diff
@@ -745,6 +745,8 @@ struct UA_DataType {
                                     pointers */
     UA_Boolean overlayable  : 1; /* The type has the identical memory layout in
                                     memory and on the binary stream. */
+    UA_UInt32  xmlEncodingId;    /* NodeId of datatype when encoded as XML */
```

Comment:
These are 64bit * 200 types -> 1,5kb added binary size.
- Currently we don't use the xmlEncodingId. The line can be commented out.
- There is no nodeid in ns0 with an identifier > 15000.  The binaryEncodingId could become an UInt16 integer.